RELEASING.md

```diff
@@ -1,10 +1,10 @@
 ### Releasing a version: 
 
-- Make a release branch `release/x.x.x`
-- Create a CHANGELOG.latest.md with the changes for the current
+- Start a git-flow release/x.y.z
+- Create a CHANGELOG.latest.md with the changes for the current version (to be used by Fastlane for the github release notes)
 - Run `fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in `android/build.gradle`, `android/gradle.properties` and `PurchasesHybridCommon.podspec`
 - Update purchases-android version in `android/build.gradle`
-- Update purchases-ios pod version in `PurchasesHybridCommon.podspec`, `ios/PurchasesHybridCommon/Podfile`, and `ios/PurchasesHybridCommon/PurchasdsHybridCommon.xcworkspace` (open this last one with Xcode)
```

Comment:
@aboedo I didn't understand what this means by "`ios/PurchasesHybridCommon/PurchasdsHybridCommon.xcworkspace` (open this last one with Xcode)". Is it required? I didn't see anything changed in the xcworkspace the other PRs updating the iOS versions

Reply:
I'm not even sure 😂  I _believe_ I meant "you should probably open it with Xcode since it's not meant to be human-readable". 
I'm glad the step is gone, though. Long live fastlane.